Makefile

```diff
@@ -11,25 +10,24 @@ PIP_REQUIRE_VIRTUALENV = 1
 endif
 export PIP_REQUIRE_VIRTUALENV
 
-default: deps
-
-deps: h.egg-info/.uptodate node_modules/.uptodate
-
-h.egg-info/.uptodate: setup.py requirements.txt
-	pip install --use-wheel -e .[dev,testing]
-	touch $@
+.PHONY: default
+default: test
 
-node_modules/.uptodate: package.json
-	$(NPM_BIN)/check-dependencies || npm install
-	touch $@
+build/manifest.json:
+	$(NPM_BIN)/gulp build
 
+.PHONY: clean
 clean:
 	find . -type f -name "*.py[co]" -delete
 	find . -type d -name "__pycache__" -delete
-	rm -f .coverage
-	rm -f node_modules/.uptodate .eggs/.uptodate
+	rm -f node_modules/.uptodate h.egg-info/.uptodate
 	rm -rf build dist
 
+.PHONY: dev
+dev: build/manifest.json h.egg-info/.uptodate
```

Comment:
Oops! An oversight on my part. Fixed. (`build/manifest.json` should have had a dependency on `node_modules/.uptodate` anyway, otherwise this wouldn't have worked on a clean checkout.